src/charm.py

```diff
@@ -309,9 +320,15 @@ def _get_k8s_ingress(self):
         if self._owasp_modsecurity_crs:
             annotations["nginx.ingress.kubernetes.io/enable-modsecurity"] = "true"
             annotations["nginx.ingress.kubernetes.io/enable-owasp-modsecurity-crs"] = "true"
+
+            nginx_modsec_file = "/etc/nginx/owasp-modsecurity-crs/nginx-modsecurity.conf"
+            logging.info(repr(self._owasp_modsecurity_custom_rules))
             annotations[
                 "nginx.ingress.kubernetes.io/modsecurity-snippet"
-            ] = "SecRuleEngine On\nInclude /etc/nginx/owasp-modsecurity-crs/nginx-modsecurity.conf"
+            ] = "SecRuleEngine On\n{}Include {}".format(
```

Comment:
Could this be an f-string?

Reply:
I suppose that would not be a problem since f-string prints '\n' correctly. I used '.format' to be consistent with the previous code. Do you think I should change the other format occurrences for f-string?

Reply:
Our guidance for the scenario is that we should always fix these issues for any code being edited updated, updating the rest of the code would be valuable but is up to the individual (e.g., it might increase the scope of the PR too much so we delay it): https://github.com/canonical/is-charms-contributing-guide/blob/main/CONTRIBUTING.md#non-compliant-code

Reply:
Fixed all, it was just about 11 strings using .format.